tutorial/lesson-1/README.md

```diff
@@ -1,17 +1,19 @@
 # Hello World Example
 
-![hello-world](https://user-images.githubusercontent.com/19663260/41165078-617d223c-6b45-11e8-8f9f-ffdc02e068ae.png)
+![hello-world-typescript](https://user-images.githubusercontent.com/19663260/41165078-617d223c-6b45-11e8-8f9f-ffdc02e068ae.png)
 
-This is an empty app used as a template for new **reSolve** applications. The `create-resolve-app` script started without arguments creates this app:
+This is an empty app used as a default template for new **reSolve** applications.
+
+To create a new TypeScript project based on this example, type the following:
 
 ```sh
-npx create-resolve-app my-app
+npx create-resolve-app hello-world-react -e react -t
```

Comment:
React template is used by default, you can just omit '-e react' in this case

Reply:
Fixed